cmd/archeio/app/handlers.go

```diff
@@ -39,6 +44,8 @@ func MakeHandler(upstreamRegistry string) http.Handler {
 		switch {
 		case strings.HasPrefix(path, "/v2/"):
 			doV2(w, r)
+		case path == "/":
```

Comment:
we should get a unit test case for this

Reply:
https://github.com/kubernetes-sigs/oci-proxy/pull/51/files